packages/storefront/packages/nuxt/CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Change log
 
+## 3.1.0
+
+- **[ADDED]** Add `middlewareModule` to `createSdk` params.
```

Comment:
we've got `createSdk` in Next.js, in Nuxt we've added the `middlewareModule` to the `defineSdkConfig` callback's params

Reply:
fixed